src/ORM/DataQuery.php

```diff
@@ -21,6 +22,15 @@
  */
 class DataQuery
```

Comment:
This refactor has the biggest chance of regressions. Anything that subclassed DataQuery would be in trouble, but that's rare and unrecommended, AFAIK.

It would be good to test the modification on sites that make a lot of use of constructing their own DataQueries, though.